client/src/components/Common/ErrorMessage.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react'
+import {View, StyleSheet, Text} from 'react-native'
+
+interface Message {
```

Comment:
Name this ErrorMessage, we already have a Message Type so we do not want to confuse them together